README.md

```
@@ -1,414 +1,493 @@
 
----
+# Check resource usage
+docker stats wazuh-mcp-server
 
-## 📄 License
+# View detailed logs
+docker compose logs wazuh-mcp-server --tail=100
+```
 
-This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
+## 💡 Usage Examples
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)